exercises/react/.meta/template.j2

```diff
@@ -0,0 +1,66 @@
+{%- import "generator_macros.j2" as macros with context -%}
+from functools import partial
```

Comment:
Use this for `ComputeCell` with a conditional `compute_function`.

Ex: `lambda inputs: {{ conditional(cell['compute_function']) }}`

```suggestion
{%- macro conditional(s) -%}
{%- set condition_start = s.find("if ") + 3 -%}
{%- set condition_end = s.find(" then ") -%}
{%- set condition = s[condition_start : condition_end] -%}
{%- set if_true_start = condition_end + 6 -%}
{%- set if_true_end = s.find(" else ") -%}
{%- set if_true = s[if_true_start : if_true_end] -%}
{%- set if_false_start = if_true_end + 6 -%}
{%- set if_false = s[if_false_start :] -%}
{{ if_true }} if {{ condition }} else {{ if_false }}
{%- endmacro -%}
from functools import partial
```

Reply:
I applied this macro and it works fine :)